src/main/scala/org/fortysevendeg/sparksftp/ReadingSFTPHadoopApp.scala

```diff
@@ -35,12 +45,23 @@ object ReadingSFTPHadoopApp extends IOApp {
       salaries = dataframeFromCSV(sparkSession, salariesPath)
 
       // Sample operations to persist and query the Hive database
```

Comment:
```suggestion
```

Reply:
Changed in: fad9c56504f0be79d73728c088ebdace33e4135b